frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ProjectActionsFormStep/ProjectActionsFormStepActions/ProjectActionsActionSelect.tsx

```diff
@@ -45,7 +45,7 @@ export const ProjectActionsActionSelect = ({
             options={actionOptions}
             autoHighlight
             autoSelect
-            value={actionOptions.find(({ key }) => key === value)}
+            value={actionOptions.find(({ key }) => key === value) || null}
```

Comment:
Is this fixing the issue? I thought that the value was unset and we're not able to set it

Reply:
Should fix the issue from some quick tests. The issue was MUI being weird to the value being `undefined`. It's OK with `null`s however, since that just means that the field is empty.